mbtools/models.py

```diff
@@ -169,6 +171,25 @@ def __init__(self, parent, location):
         self.etree = html.fromstring(self.parent.text)
         # self.etree = html.fromstring(self.parent.text)
 
+    def replace_content_tag(self):
+        # ignore replaced elements
+        attrib_dict = self.parent.attrib
+        if self.parent.tag in ["content", "contents"]:
+            if "class" in attrib_dict.keys() and \
+                    attrib_dict["class"] == "os-raise-content":
+                return {}
+            html_file_dict = {}
+            content_uuid = str(uuid.uuid4())
+            content = self.parent.text
+            tag = f'<div class="os-raise-content" ' \
+                  f'data-content-id="{ content_uuid }"></div>'
+
+            self.parent.text = escape(tag)
+            html_file_dict[content_uuid] = content
+            return html_file_dict
+        self.parent.text = escape(self.parent.text)
```

Comment:
I think this is an errant line in that the assignment shouldn't be happening as it's the code path where you are not in a `<content>` or `<contents>` and should do nothing.